scripts/ingests/21Legg_ingest.py

```diff
@@ -0,0 +1,431 @@
+from astropy.io import ascii
+from scripts.ingests.ingest_utils import *
+from astropy.utils.exceptions import AstropyWarning
+import numpy.ma as ma
+
+warnings.simplefilter('ignore', category=AstropyWarning)
+
+SAVE_DB = False  # save the data files in addition to modifying the .db file
+RECREATE_DB = True  # recreates the .db file from the data files
+db = load_simpledb('SIMPLE.db', recreatedb=RECREATE_DB)
+
+logger.setLevel(logging.INFO)
+
+
+def convert_radec_to_decimal(source):
+    # convert ra and dec to decimal string
+    ra_seconds = source['RA'][4:]
+    if float(ra_seconds) > 60.0:
+        ra_seconds = f"{source['RA'][4:6]}.{source['RA'][6:]}"
+    ra_sexg = f"{source['RA'][0:2]}h{source['RA'][2:4]}m{ra_seconds}s"
+    dec_seconds = source['Decl.'][5:]
+    if float(dec_seconds) > 60.0:
+        dec_seconds = f"{source['Decl.'][5:7]}.{source['Decl.'][7:]}"
+    dec_sexg = f"{source['Decl.'][0:3]}d{source['Decl.'][3:5]}m{dec_seconds}s"
+    coord = SkyCoord(ra_sexg, dec_sexg)
+    ra = coord.ra.degree
+    dec = coord.dec.degree
+
+    return ra, dec
+
+
+def add_publications(db):
+    ingest_publication(db, publication='Meis20.74', bibcode='2020ApJ...889...74M')  # Meisner_2020a
+    ingest_publication(db, publication='Meis20.123', bibcode='2020ApJ...899..123M')  # Meisner_2020b
+    ingest_publication(db, bibcode='2020ApJ...895..145B')
+    ingest_publication(db, bibcode='2021ApJS..253....7K')
+    ingest_publication(db, bibcode='2019ApJ...881...17M')
+    ingest_publication(db, bibcode='2021ApJ...918...11L')
+    ingest_publication(db, bibcode='2020ApJ...889..176F')
+    ingest_publication(db, publication='Pinf14.priv',
+                       description='P. Pinfield and M. Gromadzki, private communication 2014')
+
+    db.Publications.update().where(db.Publications.c.publication == 'Burn10').values(
```

Comment:
This might be nicer as a reference dictionary as in `convert_ref_name` or like we did in Sherelyn's PR?

Reply:
agreed!